### PR TITLE
docs(CSS): Update CSS Styleguide

### LINKED ISSUE #1079: Docs: CSS Class Naming Conventions
Based on [this conversation](https://github.com/rackerlabs/encore-ui/pull/1060#discussion_r33395653), we should discuss naming conventions around CSS classes for directive styling.

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rackspace.com/browse/FRMW-20

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

Closes #1079

### CODE REVIEW

**Comment by parlarjb** [comment]
LGTM

File: guides/css-styleguide.md [hunk 1]
```diff
@@ -96,15 +98,32 @@ in useful diffs and blames.
 
 In order to create consistent class names, it's encouraged to use the naming defined at [semantic-ui.com](http://semantic-ui.com). This is especially true for components that are meant to be re-usable like icons or alert boxes.
 
-## Frameworks
+## Scoping Styles to Directives
```
Comment (CITguy) [inline guides/css-styleguide.md]: This section based on conversations in #1079

**Comment by megalord** [comment]
LGTM

**Comment by glynnis** [comment]
Design: LGTM

**Pull request merged**